pkg/sip/client.go

```diff
@@ -128,11 +128,19 @@ func (c *Client) CreateSIPParticipant(ctx context.Context, req *rpc.InternalCrea
 	} else if req.RoomName == "" {
 		return nil, fmt.Errorf("room name must be set")
 	}
-	log := c.log.WithValues(
+	log := c.log
+	if req.ProjectId != "" {
+		log = log.WithValues("project", req.ProjectId)
```

Comment:
```suggestion
		log = log.WithValues("projectID", req.ProjectId)
```

Reply:
I think I've seen it spelled `project` in another repository. Will find that place and will update it too then 👍

Reply:
Done